### ISSUE TITLE
[Improvement]: Refactor AdditionalInfoPopup.tsx

### ISSUE THREAD

**Issue by elif5446** [elif5446/S390_404-Name-not-Found#254]
The additional info popup component englobes a lot of features.
The schedule feature has been extracted but the opening hours, accessibility icons and address copy features should be put in different components in the same directory in order to deflate AdditionalInfoPopup.tsx.
